python-frontend/src/test/java/org/sonar/python/tree/IPythonTreeMakerTest.java

```diff
@@ -121,14 +115,11 @@ public void line_magic_statement() {
 
   @Test
   public void system_shell_access() {
-    var parse = parseIPython("print(b)\n" +
+    var statements = parseIPython("print(b)\n" +
```

Comment:
Sorry, it wasn't clear in my previous comment, but here as well there is a test that got complexified. I think it could be nice to keep the previous test as well.